### DISCUSSION TITLE
q-tab-panels and keep-alive-include

### DISCUSSION THREAD

**Discussion by novecento** [quasarframework/quasar#10468] [General - Components / Directives / etc]
Hi,
I'm using keep-alive-include="one,two" but MyComponent is "destroyed" and I see it called beforeUnmount event. Where is my fault?

Thanks in advance.

![image](https://user-images.githubusercontent.com/1419568/130649019-314de737-fd40-4a26-9dea-1f96f3ab1e6d.png)

**Comment by ontwikkelfabriek** [comment]
@novecento 

If you create a codepen.io we can provide better support.
